### PR TITLE
Move to master branch 

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by kkraune** [comment]
should be _Copyright Verizon Media_ (and no year)

**Comment by jobergum** [comment]
Please check @kkraune and merge if you agree :)

File: vespa-cloud/cord-19-search/src/main/application/deployment.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!-- Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/src/main/application/deployment.xml]:
```suggestion
<!-- Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
```

File: vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/ApplicationMain.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/ApplicationMain.java]:
```suggestion
// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.``

File: vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/FeedAndSearchSystemTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/FeedAndSearchSystemTest.java]:
```suggestion
// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

File: vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/StagingSetupTest.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/StagingSetupTest.java]:
```suggestion
// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

File: vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/StagingVerificationTest.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/src/test/java/ai/vespa/example/cord19/StagingVerificationTest.java]:
```suggestion
// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

File: vespa-cloud/cord-19-search/scripts/add-citation-data.py [hunk 6]
```diff
@@ -0,0 +1,105 @@
+import pandas
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/scripts/add-citation-data.py]:
```suggestion
#! /usr/bin/env python3
# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.

import pandas
```

File: vespa-cloud/cord-19-search/scripts/add-citation-data.py [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+import sys
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/scripts/add-citation-data.py]:
```suggestion
#! /usr/bin/env python3
# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.

import sys
```

File: vespa-cloud/cord-19-search/scripts/add-embeddings.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
+import pandas
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/scripts/add-embeddings.py]:
```suggestion
#! /usr/bin/env python3
# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.

import pandas
```

File: vespa-cloud/cord-19-search/scripts/convert-to-feed.py [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import sys
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/scripts/convert-to-feed.py]:
```suggestion
#! /usr/bin/env python3
# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.

import sys
```

File: vespa-cloud/cord-19-search/scripts/convert-to-json.py [hunk 10]
```diff
@@ -0,0 +1,177 @@
+import pandas
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/scripts/convert-to-json.py]:
```suggestion
#! /usr/bin/env python3
# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.

import pandas
```

File: vespa-cloud/cord-19-search/scripts/convert-to-vespa.py [hunk 11]
```diff
@@ -0,0 +1,185 @@
+import pandas
```
Comment (kkraune) [inline vespa-cloud/cord-19-search/scripts/convert-to-vespa.py]:
```suggestion
#! /usr/bin/env python3
# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.

import pandas
```

**Pull request merged**